NuWeather/plugin.py

```diff
@@ -101,6 +101,9 @@ def __init__(self, irc):
             self.geocode_db = {}
         world.flushers.append(self.db.flush)
         world.flushers.append(self._flush_geocode_db)
+        self.geoback = self.registryValue('geocodeBackend')
+        if self.geoback not in GEOBACKS:
+            irc.error(_("Unknown geocode backend %s. Valid ones are: %s") % (self.geoback, ', '.join(GEOBACKS)), Raise=True)
```

Comment:
I think it would be better to fetch the geocode backend dynamically inside _geocode, instead of using a temporary variable.